packages/flutter_pptx/pubspec.yaml

```diff
@@ -16,7 +16,7 @@ dependencies:
   flutter:
     sdk: flutter
   dart_pptx: ^0.1.2+1
-  screenshot: ^1.3.0
+  screenshot: ^2.1.0
```

Comment:
The previous `screenshot` package was providing the deprecated `window` property to the `RenderView` class, the new version of the `screenshot` package handles this issue.